tests/ImageSharp.Tests/TestUtilities/Attributes/Intrinsics/IntrinsicTheoryAttribute.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using Xunit;
+
+namespace SixLabors.ImageSharp.Tests
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public class IntrinsicTheoryAttribute : TheoryAttribute
```

Comment:
`RuntimeFeatureConditionalTheory` ?

Reply:
Your naming is more clear to undestand considering `RuntimeFeatureFilter` enum naming but isn't it too long?

Reply:
Yeah it's super verbose, but it's in sync with arcade's `[ConditionalTheory]` which we already use. Maybe @JimBobSquarePants  has some suggestions.

Reply:
@JimBobSquarePants sorry for namedropping but can you give your opinion on this?